astropy/modeling/mappings.py

```diff
@@ -10,16 +10,34 @@
 
 
 class Mapping(Model):
-    def __init__(self, mapping):
+    """
+    Allows inputs to be reordered, duplicated or dropped.
```

Comment:
By itself this doesn't tell much about how to use it (even with the examples).  I don't think it should go overboard though--I like this as it is (short and sweet), but maybe it should link to the "Advanced Mappings" section in the documentation for more info.